fe/fe-core/src/main/java/com/starrocks/authentication/UserProperty.java

```diff
@@ -27,27 +27,42 @@ public class UserProperty {
     @SerializedName(value = "m")
     private long maxConn = 1024;
 
-    private static final String PROP_MAX_USER_CONNECTIONS = "max_user_connections";
+    @SerializedName(value = "mi")
+    private long maxIpConn = 1024;
+
+    public static final String PROP_MAX_USER_CONNECTIONS = "max_user_connections";
+
+    public static final String PROP_MAX_USER_IP_CONNECTIONS = "max_user_ip_connections";
```

Comment:
I'd like to suggest the name is max_connections_per_ip

Reply:
ok，I will be modify it later